lib/plugins/aws/provider.js

```diff
@@ -304,6 +304,10 @@ class AwsProvider {
             },
             additionalProperties: false,
           },
+          awsApiGatewayPayloadVersion: {
```

Comment:
let's call it `awsHttpApiPayload` to keep the naming consistent with where it will be used

Reply:
Also, let's use that definition inside `httpApi.payload` schema definition as it should be using the same one